app/models/user.rb

```diff
@@ -0,0 +1,19 @@
+class User < ApplicationRecord
+  before_save :downcase_email
+
+  validates :name, presence: true, length: {maximum: Settings.users[:max_name]}
```

Comment:
```suggestion
  validates :name, presence: true, length: {maximum: Settings.users.max_name}
```
Tương tự vs mấy cái bên dưới nhé